jekyll/_cci2/artifacts.md

```diff
@@ -131,9 +131,9 @@ Finally, the core dump files are stored to the artifacts service with `store_art
 
 ![Core Dump File in Artifacts Page]( {{ site.baseurl }}/assets/img/docs/core_dumps.png)
 
-When CircleCI runs a build, a link to the core dump file appears under the Artifacts tab of the {% comment %} TODO: Job {% endcomment %}build summary.
+When CircleCI runs a build, a link to the core dump file appears in the Artifacts tab of the **Jobs page**.
```

Comment:
"When CircleCI builds a job"

Reply:
Or "runs a job". I'm not sure it matters.

Reply:
I think "runs" a job is less confusing. "Build" already has some other meanings, so it'll be easier to grok if we leave the word out entirely.